README.md

```diff
@@ -1,2 +1,135 @@
+
+The Festival voice can be used by supplying `-eval <voice_dir>/festvox/<voicename>_clunits.scm -eval '(voice_<voicename>_clunits)'` when invoking festival binaries such as `${FESTIVALDIR}/bin/text2wave`. Note that `text2wave` also expects a path to a file containing the text to be synthesized as a required argument, and output can be redirected to a file using the `-o` flag. Example where input.txt contains a text prompt:
+
+```
+$FESTIVALDIR/bin/text2wave -eval lvl_is_f/festvox/lvl_is_f_clunits.scm -eval '(voice_lvl_is_f_clunits)' input.txt -o output.wav
+```
+
+The generated flite voice binary accepts a sequence of phonemes as input, using the `-p` flag. e.g. 
+
+```
+./flite_lvl_is_f -p "C E1 r t n a0 EE1 rz s E1 h t n i0 N k pau s EE1 m T uu1 c aii1 t I0 r n OO1 t a0 T pau" -o output.wav
+```
+Called from within the `<voice_dir>/flite` subdirectory after the voice build, this should generate a synthetic clip of a voice saying "Hérna er setning sem þú gætir notað" in the file `output.wav`.
+
+
+The word-level token for silence is '<sil>' whereas the phone-level token is 'pau'.
+
+## Known issues and further development
+The voices resulting from this recipe are by no means SOTA voices and have significant limitations, particulary in terms of audible joins. This issue can be caused by 3 potential causes.
+- Firstly, the pitch marks (extracted into the `<voice_dir>/pm` subdirectory) may be incorrectly computed and the extraction script `<voice_dir>/bin/make_pm_wave` may need tweaking for that particular voice.
+- Secondly, the join cost function may have suboptimal parameters. In the festival voice, they are set in `<voice_dir>/festvox/<voice_name>_clunits.scm` in the case of runtime parameters and in `<voice_dir>/festvox/build_clunits.scm` in the case of build-time parameters, as the `<voice_name>::dt_params` variable.
+  - In particular, the parameters of note here are `f0_join_weight` and `f0_pen_weight`, which denotes the weighting placed upon mismatches in pitch, and `join_weights` and `ac_weights`, which correspond to mismatches in `f0` and `MFCC` features on joins. `MFCCs` roughly correspond to the overall shape of the spectrum at the join points. 
+  - Other parameters can also be tweaked, they are documented on pages 116-122 of [the manual for building synthetic voices by Alan W. Black and Kevin A. Lenzo](http://festvox.org/bsv/bsv.pdf)
+  - In Flite, these parameters are stored in the `<voicename>_db` variable which is set near the bottom of `flite/<voicename>_clunits.c`
+- Thirdly, the waveform join method may be suboptimal. That is set by the `join_method` parameter in the runtime parameters of Festival. 
+  - Supported values in festival are *windowed*, *none* and *modified_lpc*.
+  - In Flite, only *simple_join*, which corresponds to *none* and *modified_lpc* are implemented, and they are specified in the `register_<voicename>` function in `flite/<voicename>.c`, along with other parameters. It can be set by e.g.:
```

Comment:
maybe some words about the number of wav. files used for the generation of the voice and some intuition about how the number can be tweaked and what the expected effects in terms of qualitiy vs. voice size means ?